### DISCUSSION TITLE
Add support for sqlite3 strict mode (disable legacy double-quoted string literal misfeature)

### DISCUSSION THREAD

**Discussion by shaicoleman** [jeremyevans/sequel#2233] [General]
The `sqlite3` gem allows to set `strict: true` when opening a database. This disables `SQLITE_DBCONFIG_DQS_DDL`, `SQLITE_DBCONFIG_DQS_DML`, which allow to disable the legacy double-quoted string literal misfeature.

Currently there doesn't seem a way to do the same in Sequel.

```ruby
require 'sqlite3'
require 'sequel'

# Disable SQLITE_DBCONFIG_DQS_DDL, SQLITE_DBCONFIG_DQS_DML
sqlite3 = SQLite3::Database.new(':memory:', { strict: true })
begin
  sqlite3.execute('SELECT "1"')&.first
rescue SQLite3::SQLException => e
  # Prints no such column: "1" in strict mode
  puts "sqlite3 gem: #{e.message}"
end

sequel = Sequel.connect('sqlite:/', strict: true, driver_options: { strict: true })
result = sequel.execute('SELECT "1"')&.first
puts "sequel gem: #{result}"
```

**Comment by jeremyevans** [comment]
I agree that this feature seems like a good idea.  Is this something you would like to work on implementing?  The actual change should be possible in a single line, though the testing code would definitely require more than one line.

**Comment by shaicoleman** [comment]
@jeremyevans, I think you'll be in a better position to implement it, and I'd appreciate it if you would do it, thanks

**Reply by jeremyevans** [reply]
Sure, I should be able to make the change before the next release.  Thank you for suggesting this feature!

**Reply by jeremyevans** [reply]
I implemented this in 8e5a8a92cadfa36ebec01c70340ef8e8ee1feff6, though I changed the option name to :disable_dqs to accurately reflect what the option does.
